tests/ArgentPonyWarcraftClient.Integration.Tests/TestUtilities/RequestResultExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using System.Text.Json;
+using FluentAssertions.Json;
+using Newtonsoft.Json.Linq;
+
+namespace ArgentPonyWarcraftClient.Integration.Tests
+{
+    public static class RequestResultExtensions
```

Comment:
Class is definitely copy-pasted between the integration tests and unit tests. Should clean this up into some kind of reusable class/dll so that it's shared between the two test projects instead of copy-pasted.

Reply:
As long as there is only a small amount of duplication, I consider it preferable to having one test project reference the other or introducing another project just for a handful of shared types for tests.